contents/spotlight/startup-puzzle.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "How Puzzle defined a buyer persona and built around it"
+date: 2024-02-01
+author: ["joe-martin"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/startup_puzzle.jpg
+featuredImageType: full
+category: Startups
+---
+
+> Each month, we speak to a team enrolled in PostHog for Startups about the way they work. This time, we spoke to [Puzzle](https://puzzle.io/?utm_medium=newsletter&utm_source=posthog_newsletter&utm_campaign=posthog_startupfeature), an accountancy platform for early-stage startups, about how they define and use their buyer persona. 
+
+Building a persona for your ideal or target customers is one of the most challenging but essential steps for growing your product. Not only will it help you make product and pricing decisions, it will also clarify your value proposition and dictate how you judge successes and failures.
+
+We’ve previously documented [how we defined our personas](/founders/creating-ideal-customer-profile), but not all companies approach the task in the same way. So, we spoke to [Puzzle's](https://puzzle.io/?utm_medium=newsletter&utm_source=posthog_newsletter&utm_campaign=posthog_startupfeature) Scott Cormnier, Head of Product (Growth & Monetization) about how the team defined their persona, what impact the definition had, and what recommendations they had for others...
+
+## 1. Collect both qualitative and quantitative data
+
+A persona is, fundamentally, a snapshot of your ideal customer based on a broad understanding of many customers. In order to define your persona you therefore need to start by speaking to your existing users directly and building an intuitive understanding of their needs.
+
+Once you’ve done that, says Scott, you need to pair that intuitive understanding with quantitative data, so you can understand if these behaviors exist in isolation or at scale. 
+
+“We’re constantly standing next to founders at each stage of the startup journey interviewing and learning about their experience,” says Scott. “But it’s one thing for a customer to say they want, need, or plan to do X, Y, or Z — and another thing if they actually do it. That’s where quantitative data is useful. You need both to create your ICP.”
```

Comment:
I feel a bit confused if we introduce ICP as a concept to this article, can we just use persona?
```suggestion
“We’re constantly standing next to founders at each stage of the startup journey interviewing and learning about their experience,” says Scott. “But it’s one thing for a customer to say they want, need, or plan to do X, Y, or Z — and another thing if they actually do it. That’s where quantitative data is useful. You need both to create your persona.”
```